libraries/core_libs/consensus/include/dag/dag_block_manager.hpp

```diff
@@ -90,6 +98,10 @@ class DagBlockManager {
   std::condition_variable_any cond_for_verified_qu_;
   std::map<uint64_t, std::deque<DagBlock>> verified_qu_;
 
+  std::atomic_uint64_t dag_expiry_level_ =
```

Comment:
There is another dag_expiry_level_ in dag.hpp. Does this duplicate?

Reply:
See my answer to Matus